src/ui/page/file/view.jsx

```diff
@@ -147,6 +148,45 @@ class FilePage extends React.Component<Props> {
       showToast,
     } = this.props;
 
+    const claimAuthor = `${claim.value.author} (${claim.channel_name})`;
```

Comment:
We don't really expose author even though it's available - it should use channel if there is one (could be anonymous too)